tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -114,7 +126,11 @@ def test_single_partition(self, metadata_quorum=quorum.zk):
         self.logger.info("TASKS: %s\n" % json.dumps(tasks, sort_keys=True, indent=2))
 
     @cluster(num_nodes=10)
-    @matrix(metadata_quorum=quorum.all_non_upgrade)
+    @matrix(
+        metadata_quorum=quorum.all_non_upgrade,
+        se_new_coordinator=[True, False]
+    )
+    @skip_if_new_coordinator_and_zk
     def test_multiple_consumers_random_group_topics(self, metadata_quorum=quorum.zk):
```

Comment:
Is having `use_new_coordinator` as a param optional?

Reply:
no, sry again the changes got missed